e2e-tests/insurance/cypress/e2e/journeys/eligibility/change-your-answers/change-your-answers-uk-goods-or-services.spec.js

```diff
@@ -0,0 +1,80 @@
+import { INSURANCE_ROUTES } from '../../../../../../constants/routes/insurance';
+import { INSURANCE_FIELD_IDS } from '../../../../../../constants/field-ids/insurance';
+import { completeAndSubmitBuyerCountryForm } from '../../../../../../commands/forms';
+import { summaryList, noRadio, submitButton } from '../../../../../../pages/shared';
+
+const { HAS_MINIMUM_UK_GOODS_OR_SERVICES } = INSURANCE_FIELD_IDS.ELIGIBILITY;
+
+const {
+  START,
+  ELIGIBILITY: { UK_GOODS_OR_SERVICES_CHANGE, CHECK_YOUR_ANSWERS, CANNOT_APPLY },
+} = INSURANCE_ROUTES;
+
+const baseUrl = Cypress.config('baseUrl');
+
+context('Insurance - Eligibility - Change your answers - UK goods or services - As an exporter, I want to change my answers to the eligibility UK goods or services section', () => {
+  const url = `${baseUrl}${CHECK_YOUR_ANSWERS}`;
+
+  before(() => {
+    cy.navigateToUrl(START);
+
+    cy.completeStartForm();
+    cy.completeCheckIfEligibleForm();
+    cy.completeExporterLocationForm();
+    cy.completeCompaniesHouseNumberForm();
+    cy.completeAndSubmitCompaniesHouseSearchForm({});
+    cy.completeEligibilityCompanyDetailsForm();
+    completeAndSubmitBuyerCountryForm({});
+    cy.completeAndSubmitTotalValueInsuredForm({});
+    cy.completeCoverPeriodForm({});
+    cy.completeUkGoodsAndServicesForm();
+    cy.completeEndBuyerForm();
```

Comment:
These commands are repeated a lot, how about a new command? 

```js
cy.completeAllEligibilityAnswers({ });
```

We need to do this for other specific eligibility questions, but this is for all questions so makes sense to be a new command IMO